apps/files_versions/lib/Listener/FileEventsListener.php

```diff
@@ -402,6 +403,24 @@ private function getPathForNode(Node $node): ?string {
 			}
 		}
 
+		if (!($node instanceof NonExistingFile) && !($node instanceof NonExistingFolder)) {
+			$this->logger->debug('Failed to compute path for node', [
+				'node' => [
+					'path' => $node->getPath(),
+					'owner' => $owner,
+					'fileid' => $node->getId(),
+					'size' => $node->getSize(),
+					'mtime' => $node->getMTime(),
+				]
+			]);
+		} else {
+			$this->logger->debug('Failed to compute path for node', [
+				'node' => [
+					'path' => $node->getPath(),
+					'owner' => $owner,
+				]
+			]);
+		}
```

Comment:
Kind of a nitpick, but this could be shortened to:

```suggestion
		$node = [
			'path' => $node->getPath(),
			'owner' => $owner,
		];
		$nodeInfo[]['node'] = $node;
		if (!($node instanceof NonExistingFile) && !($node instanceof NonExistingFolder)) {
			$node['fileid'] = $node->getId();
			$node['size'] = $node->getSize();
			$node['mtime'] = $node->getMTime();
		}
		$this->logger->debug('Failed to compute path for node', $nodeInfo);
```

Reply:
too late to the party :eyes: